ppr-ui/src/utils/form-helper.ts

```diff
@@ -0,0 +1,12 @@
+export function focusOnFirstError (formName: string): void {
+  setTimeout(() => {
+    const firstErrorField = document
+      .getElementById(formName)
+      .getElementsByClassName('error--text')
+      .item(0)
+      .getElementsByTagName('input')
+      .item(0) as HTMLElement | null
+
+    firstErrorField.focus()
+  }, 50)
+}
```

Comment:
This was the UX addition you mentioned?

Reply:
Yes, this is the ability to jump/focus on the first error. Might need to polish a bit and remove form parameter (if designers will like the feature).